PocketKit/Sources/PocketKit/Root/RootCoordinator.swift

```diff
@@ -41,6 +42,64 @@ class RootCoordinator {
         }.store(in: &subscriptions)
 
         window?.makeKeyAndVisible()
+        
+        rootViewModel.$bannerViewModel.sink { [weak self] viewModel in
+            if let viewModel = viewModel {
+                self?.setupBanner(with: viewModel)
+            } else {
+                self?.dismissView()
+            }
+        }.store(in: &subscriptions)
+
+        NotificationCenter.default.publisher(for: UIApplication.didEnterBackgroundNotification).sink { [weak self] _ in
+            self?.removeBanner()
```

Comment:
What do you think about moving this into the root view model? We could listen for this same notification and then set `self.bannerViewModel` to nil to dismiss. I think it gives a nice symmetry with the logic that presents the banner (and it also moves this business logic out of the coordinator)